Sources/generate-symbol-graph/main.swift

```diff
@@ -593,7 +619,20 @@ let symbols: [SymbolGraph.Symbol] = supportedDirectives.map { directive in
         mixins: [
             SymbolGraph.Symbol.DeclarationFragments.mixinKey: SymbolGraph.Symbol.DeclarationFragments(
                 declarationFragments: fragments
-            )
+            ),
+            SymbolGraph.Symbol.Availability.mixinKey: SymbolGraph.Symbol.Availability(availability: [
+                .init(
+                    domain: .init(rawValue: "Swift-DocC"),
```

Comment:
Nice! Did you consider using "Swift" instead of "Swift-DocC" here? My mental model is that we don't actually have releases of Swift-DocC – Swift-DocC is just included in releases of Swift.

Reply:
I did consider it and I can change it if you prefer. 

The only thing I wanted to avoid was someone misunderstanding "Swift 5.9+" and thinking that they can't use it because their package tools version is 5.8. It only matters what toolchain they have, but I'm not sure how to best convey that.

Reply:
Ah right. Great point. This reminds of a discussion on the forums I saw recently – we don't currently have a way to indicate "this API was introduced in Swift 5.9 but is back-ported to Swift 5.6 and so as long as you build with Swift 5.9, you can target version Swift 5.6 and use this symbol."

I think it makes since to use "Swift-DocC" at least until we solve that problem. Thanks!